cli/project/files.go

```diff
@@ -281,6 +282,12 @@ func SetLocalRunningPaths(ctx *context.Ctx) error {
 	return nil
 }
 
+// GetDefaultLocalRunningPaths returns a list of default
+// local paths RunDir, DataDir, LogDir.
+func GetDefaultLocalRunningPaths() []string {
```

Comment:
Why "Get**Default**LocalRunningPaths" ? AFAIU we can return one of the default or value from config as in "SetLocalRunningPaths". (If you do so, remove "tmp..." from commit title and changelog).

Reply:
No longer relevant